client/src/components/userForm/userForm.js

```diff
@@ -1,8 +1,9 @@
 import React from "react";
+import Logout from '../Logout/Logout';
 import "../../../node_modules/bootstrap/dist/css/bootstrap.min.css";
 import { BrowserRouter as Router, Switch, Route } from "react-router-dom";
 import Login from "../login-signup/Login";
-import SignUp from "../login-signup/Signup";
+import Register from "../login-signup/Signup";
```

Comment:
Why has the name changed? Should we also change the name of the file from Signup.js to Register.js to be consistent?

Reply:
will change with the next pull request